schedule/yast/select_modules_and_patterns+registration/select_modules_and_patterns+registration@s390x.yaml

```diff
@@ -18,10 +18,12 @@ schedule:
   - installation/bootloader_start
   - installation/setup_libyui
   - installation/access_beta_distribution
-  - installation/licensing/accept_license
-  - installation/registration/skip_registration
-  - installation/module_selection/select_nonconflicting_modules
-  - installation/add_on_product_installation/accept_add_on_installation
+  registration:
```

Comment:
you cannot mix list of modules with keys to overwrite, both approach are do not coexist.
Please, follow the recommendations below for a s390x kvm default file, and overwrite everything that doesn't match.
That I can see at first view:
- Overwrite `registration` for not registering the system (we usually will do that in Online). This is probably a good use of a 'flow' so we can have the test cases for both Online and Full and just adjust the flow per each, but you don't need to achieve that perfect here, you can just overwrite if you prefer, the current flow create I'm not sure if it is correct.
- A flow for desktop can be used, but not really needed, you can just overwrite.
- `software` will need to be overwriten for selecting the patterns.
- Overwrite `system_preparation` with `console\{hostname,system_prepare,force_scheduled_tasks}` and `grub_set_bootargs`. and `system_validation` with the rest.